README.md

```diff
@@ -1,3 +1,58 @@
 # Github Actions for Zenodo
+Documentation for each action could be found in their respective directories.
 
-Documentation for each action could be found in their respective directories.
\ No newline at end of file
+## Create a Record 
```

Comment:
Is this README supposed to be on the root of the folder?

Reply:
Create Record is the combination of all the actions, and there is no dedicated directory for that, so to give details about the workflow for publishing a record I have to do that in the root readme. You can tell me if it needs to be somewhere else.

Reply:
I see. So this is showing a complete example. Yeah, this makes sense.